src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -142,11 +142,9 @@ module Iostat = struct
 
     (* Now read the values out of dev_values_map for devices for which we have data *)
     Listext.List.filter_map (fun dev ->
-        if not (Hashtbl.mem dev_values_map dev)
-        then None
-        else
-          let values = Hashtbl.find dev_values_map dev in
-          Some (dev, values)
+        match Hashtbl.find_opt dev_values_map dev with
+        | None -> None
+        | Some values -> Some (dev, values)
```

Comment:
`option` is a functor, so you can map:

`Hashtbl.find_opt dev_values_map dev |> Option.map (fun v -> (dev, v))`

Reply:
> Since 4.08
;(